### PR TITLE
Use PHPUnit 8 for all packages

### PR DESCRIPTION
**Pull request by greg0ire**
Some proof PRs:

- https://github.com/sonata-project/SonataPageBundle/pull/1105
- https://github.com/sonata-project/sonata-doctrine-extensions/pull/164
- https://github.com/sonata-project/SonataNotificationBundle/pull/408

### CODE REVIEW

**Comment by OskarStark** [comment]
Doctrine extensions and page bundle?

**Comment by greg0ire** [comment]
Yes, I amended my commit, but forgot the title and the description :sweat_smile:

**Comment by greg0ire** [comment]
I added one more.

**Comment by core23** [comment]
Should we require PHPUnit 8 for every project and fix the problems afterwards?

EOL for PHPUnit8 is next week: https://phpunit.de/supported-versions.html

**Comment by greg0ire** [comment]
Ok let's try this :crossed_fingers:

File: project/.travis.yml.twig [hunk 1]
```diff
@@ -41,7 +41,7 @@ env:
     - SYMFONY_DEPRECATIONS_HELPER=max[self]=0
     - TARGET=test
     - UPSTREAM_URL=https://github.com/sonata-project/{{ repository_name }}.git
-    - PHPUNIT_VERSION={{ phpunit_version|default('7') }}
+    - PHPUNIT_VERSION={{ phpunit_version|default('8') }}
```
Comment (core23) [inline project/.travis.yml.twig]: Can you remove the old `phpunit_version` settings from the project configuration?

**Pull request merged**